File: Backend/MptRoomAPI/MptRoomAPI/DTO/PostDTO.cs
using MptRoomAPI.Models.Enums;

namespace MptRoomAPI.DTO
{
    public class PostDTO
    {
        public int? PostId { get; set; }
        public string PostTitle { get; set; }
        public string PostDescription { get; set; }
        public PostTypeEnum PostType { get; set; }
        public int PostThemeId { get; set; }
        public int CourseId { get; set; }
        public int UserId { get; set; }
        public DateTime Created { get; set; }
    }
}

File: Frontend/MptRoomFrontend/src/app/models/DTO/post.dto.ts
import { PostTypeEnum } from '../enums/post-type.enum';

export interface PostDTO {
  postId?: number;
  postTitle: string;
  postDescription: string;
  postType: PostTypeEnum;
  postThemeId: number;
  courseId: number;
  userId: number;
  created: string;
}
